src/consts.rs

```diff
@@ -18,3 +18,5 @@ pub const EULER_MASCERONI: f64 = 0.577_215_664_901_532_9;
 pub const LN_LN_2: f64 = -0.366_512_920_581_664_3;
 /// ln(2 * pi * e)
 pub const LN_2PI_E: f64 = 2.837_877_066_409_345_3;
+
+pub const TWO_PI: f64 = std::f64::consts::TAU;
```

Comment:
Oh nice. They've thought of everything.

Reply:
Yeah for a minute I just used `TAU` all over the place, but it's a little confusing